pkg/compare/parsing.go

```diff
@@ -179,21 +192,27 @@ func parseDiffConfig(filePath string) (UserConfig, error) {
 
 const noValue = "<no value>"
 
-func executeYAMLTemplate(temp *template.Template, params map[string]any) (*unstructured.Unstructured, error) {
+func executeYAMLTemplatRaw(temp *template.Template, params map[string]any) ([]byte, error) {
```

Comment:
i think there is no need for this function anymore,  all of the usages are with the executeYAMLTemplate function